resources/js/store/store-supporter-tag.tsx

```diff
@@ -223,16 +229,15 @@ export default class StoreSupporterTag extends React.Component<Props> {
 
   @action
   private readonly getUser = (username: string) => {
-    this.xhr = $.ajax({
-      data: { username },
-      dataType: 'json',
-      type: 'POST',
-      url: route('users.check-username-exists'),
-    });
+    this.xhr = apiLookup([username]);
 
     this.xhr
-      .done((data) => runInAction(() => {
-        this.user = data;
+      .done((response) => runInAction(() => {
+        if (response.users.length > 0) {
+          this.user = response.users[0];
+        } else {
+          this.user = userNotFoundJson;
```

Comment:
the response isn't an observable so might as well just `response.users[0] ?? userNotFoundJson`?